docs/tools.md

```
@@ -0,0 +1,1113 @@
+# Tools
+
+> Expose functions as executable capabilities for your MCP client.
+
+export const VersionBadge = ({version}) => {
+  return <code className="version-badge-container">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: code

(MD033, no-inline-html)